verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -243,41 +241,35 @@ function RowMenu(props: {
     );
 }
 
-const downloadFilename = `globalhealth_covid19_cases_${renderDate(
-    new Date(),
-)}.csv`;
-
 export function DownloadButton(props: { search: string }): JSX.Element {
-    const [downloading, setDownloading] = React.useState(false);
-
-    const downloadCases = (): void => {
-        const requestBody = props.search.trim() ? { query: props.search } : {};
-        setDownloading(true);
-        axios
-            .post('/api/cases/download', requestBody)
-            .then((response) => fileDownload(response.data, downloadFilename))
-            .catch((e) => console.error(e))
-            .finally(() => setDownloading(false));
-    };
+    const formRef: RefObject<any> = React.createRef();
 
     return (
-        <Button
-            variant="outlined"
-            color="primary"
-            onClick={downloadCases}
-            disabled={downloading}
-            startIcon={
-                downloading ? <CircularProgress size={20} /> : <SaveAltIcon />
-            }
-        >
-            Download
-        </Button>
+        <>
+            <form
+                hidden
+                ref={formRef}
+                method="POST"
+                action="/api/cases/download"
+            >
+                <input name="query" value={props.search.trim()} />
+            </form>
+            <Button
+                variant="outlined"
+                color="primary"
+                onClick={(): void => formRef.current.submit()}
+                startIcon={<SaveAltIcon />}
+            >
+                Download
```

Comment:
we can't get the loading spinner anymore?

Reply:
It's not needed anymore as the browser download object shows up quickly (the bottom panel in chrome) and shows the loading there.

Reply:
nice! makes sense.